docs/content/preview/faq/operations-faq.md

```diff
@@ -145,35 +147,37 @@ This confirms that the password for `John` will expire in approximately 4 hours.
 
 ### What exactly does the upgrade with rollback feature do?
 
-The feature upgrades the binaries and enables new features that do not change the data format on disk. Features that do require changes to the format of data sent over the network, or stored on disk, are not enabled until you finalize the upgrade.
+The feature upgrades the binaries and enables new features that do not change the data format on disk. Features that do require changes to the format of data sent over the network, or stored on disk, are disabled until you finalize the upgrade.
+
+Because the vast majority of new changes do not modify data format, and most changes that do are from new features that are not yet in use, this temporary disabling typically has no impact on current operations. This allows you to monitor the new binary for a while (validating about 90% of code changes) to make sure there is no impact to your application and rollback to the previous version if needed.
```

Comment:
```suggestion
Because the vast majority of new changes do not modify the data format, and most changes that do are from new features that are not yet in use, this temporary disabling typically has no impact on current operations. This allows you to monitor the new version for a while (validating typically about 90% of code changes) to ensure the upgrade has not impacted your application.
```